can/strings_to_candata.cc

```diff
@@ -0,0 +1,37 @@
+#include <capnp/serialize.h>
+
+#include <string>
+#include <vector>
+
+#include "cereal/gen/cpp/log.capnp.h"
+#include "opendbc/can/common.h"
+
```

Comment:
I think we want this to go in openpliot. opendbc should be completely independent of cereal, i.e. `grep cereal` should have no hits in opendbc/